modules/exploits/windows/local/ms16_032_secondary_logon_handle_privesc.rb

```diff
@@ -0,0 +1,519 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+require 'msf/core/payload_generator'
+require 'rex'
+
+class MetasploitModule < Msf::Exploit::Local
+  Rank = GoodRanking
+
+  include Msf::Exploit::Powershell
+  include Msf::Post::File
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'          => 'MS16-032 Secondary Logon Handle Privilege Escalation',
+      'Description'   => %q{
+This module exploits the lack of sanitization of standard handles in Windows' Secondary Logon Service.  The vulnerability is known to affect versions of Windows 7-10 and 2k8-2k12 32 and 64 bit.  This module will only work against those versions of Windows with Powershell 2.0 or later and systems with two or more CPU cores. Works about 75% of the time, if module fails, just re-run.
+
+ For further information, please visit: https://googleprojectzero.blogspot.co.uk/2016/03/exploiting-leaked-thread-handle.html
+
+Copyright 2016, Ruben Boonen (@FuzzySec)
+License BSD 3-Clause
+          },
+       'License'       => "BSD 3-Clause",
```

Comment:
License should match a constant, like 'License' => BSD_LICENSE (which is probably what you want, but we can add a 3-clause specifically if that isn't what it points to today).